src/components/books.jsx

```diff
@@ -0,0 +1,60 @@
+import { NavLink } from 'react-router-dom';
+import '../styles/books.css';
+import PropTypes from 'prop-types';
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] [OPTIONAL] instead of disabling linters kindly remove `modifybooks` as it is not used